gradle.properties

```diff
@@ -1,5 +1,9 @@
 org.gradle.parallel=true
 org.gradle.caching=true
 
+# limiting the number of parallel workers helps to keep your machine responsive during builds
+# see https://github.com/gradle/gradle/issues/14224
+org.gradle.workers.max=2
```

Comment:
Just for reference, I have 4 cores with hyperthreading, and max-workers of 4 works well since it ends up with 8. This will end up with 6 workers if I'm not mistaken (build always adds 4 extra workers) - wonder if it works on a 4-core without hyperthreading machine.